CalendarApp/View Controllers/ComposeNoteViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  ComposeNoteViewController.swift
+//  CalendarApp
+//
+//  Created by Angelina Zhang on 8/2/22.
+//
+
+import Foundation
+
+protocol ComposeNoteDelegate: NSObject {
+    func didTapBack(note: Note, index: Int?);
+    func didUpdateNote(note: Note, index: Int);
+    func didDelete(noteID: UUID);
+    func didCreateNewNote(note: Note);
+}
+
+class ComposeNoteViewController : UIViewController, UITextViewDelegate, UITextFieldDelegate {
+    public var isNewNote = false
```

Comment:
It's weird to me that these properties are public. I need to dig in a little more and see if there's a good reason but this strikes me as off.